v2/community/migration/migrating-supertokens.mdx

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
this page needs to be there in each of the recipe guides as well - as a separate section, between the existing Migration and References section. Also, rename the existing Migration section to "Migrating to SuperTokens"